### PR TITLE
Support Micrometer @MeterTag

### PR DESCRIPTION
**Pull request by computerlove**
Micrometer defines a annotation @MeterTag, making it possible to add additional tags for methods annotated with @Counted and @Timed from method arguments.

### CODE REVIEW

**Comment by computerlove** [comment]
I have made a suggestion for how to support `@MeterTag`. 

MeterTag.resolver is supported, the application can define beans imlementing `ValueResolver` and refere to these classes in MeterTag.resolver=ValueResolverBean.class.

MeterTag.expression is also supported, the application can define a bean imlementing `ValueExpressionResolver` and it will be used to evaluate the expression.

**Comment by brunobat** [comment]
There is an error in the docs that needs fixing

**Comment by computerlove** [comment]
It's now fixed.

**Review by gsmet — DISMISSED** [review]
Just a note that this will need a squash before merging. Please dismiss when done.

**Comment by computerlove** [comment]
Commits has been squashed.

**Review by brunobat — COMMENTED** [review]
@computerlove would you be able to add an integration test to `integration-tests/micrometer-prometheus`?... Mimicking the usage you have in the docs would be perfect.

**Comment by computerlove** [comment]
> @computerlove would you be able to add an integration test to `integration-tests/micrometer-prometheus`?... Mimicking the usage you have in the docs would be perfect.

Yes, will do!

**Comment by computerlove** [comment]
Are the updated tests good enough?

I have been thinking about supporting `MeterTag.expression` with Qute(if present), validating the templates on build time. 
But that will have to be in a separate PR, I don't have any more spare time to work on this right now.

File: extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/MicrometerTimedInterceptor.java [hunk 1]
```diff
@@ -1,9 +1,6 @@
 package io.quarkus.micrometer.runtime;
 
-import java.lang.reflect.Method;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```
Comment (ebullient) [inline extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/MicrometerTimedInterceptor.java:3]: preserve original imports

**Review by ebullient — CHANGES_REQUESTED** [review]
rebase and squash. This looks good, just some consistency changes.

**Comment by computerlove** [comment]
I have fixed the comments for documentation and imports.

**Review by ebullient — APPROVED** [review]

**Comment by ebullient** [comment]
@gsmet .. I think we're good to go.

**Pull request merged**